packages/appeals-service-api/src/services/responses.service.js

```diff
@@ -1,8 +1,11 @@
 const ApiError = require('../errors/apiError');
 const logger = require('../lib/logger');
 const { ResponsesRepository } = require('../repositories/responses-repository');
-
 const responsesRepository = new ResponsesRepository();
+const { HasQuestionnaireMapper } = require('../mappers/questionnaire-submission/has-mapper');
+const questionnaireMapper = new HasQuestionnaireMapper();
+const { LocalEventClient } = require('@pins/common/src/event-client/local-event-client');
```

Comment:
This will need to use the service bus client. There's an example of how back office do this here, they basically allow you to configure if it should use the service bus client or the local at run time https://github.com/Planning-Inspectorate/back-office/blob/main/packages/event-client/src/event-client.js. I would suggest we default to true for using the service bus client then set false locally on the docker compose file for the appeals service api.